SocialFilter/sections/css_extended.txt

```diff
@@ -2,6 +2,9 @@
 ! Extended CSS rules
 !
 singleclickapps.com#?#.nav-list > li.nav-header:contains(/\sUs on\s/)
+geo.de##.social-bar > dd:has(> a.fb-button)
+geo.de##.social-bar > dd:has(> a.twitter-button)
+geo.de##.sidebar > section.sidebar-teasergroup:has(> iframe[src^="https://www.facebook.com/"])
```

Comment:
All three should normally be #?# instead of ## - but imho we should just add ##.social-bar to the site, nobody shares via mail.
If we should leave the mail one, use this rule instead of the first two:
.social-bar > dd > a:not(.mail-button)

Last element "iframe[src^="https://www.facebook.com/"]" not shown.

Reply:
<details>
<summary>Facebook widget leftover <a href="https://www.geo.de/reisen/reisewissen/22694-rtkl-international-ocean-film-tour-brechenden-wellen-vergisst-dean-dass-er">here</a></summary>
<img src="https://user-images.githubusercontent.com/18536328/75992353-e1bf2900-5f08-11ea-9150-ac0c6be25ceb.png" />
</details>

Reply:
> All three should normally be #?# instead of ## - but imho we should just add ##.social-bar to the site, nobody shares via mail.
> If we should leave the mail one, use this rule instead of the first two:
> .social-bar > dd > a:not(.mail-button)

There are pinterest link and link to their facebook page in `.social-bar` too

Reply:
I see the text, but not the iframe anymore.
Better switch to
.sidebar-teasergroup:has(> .sidebar-headline:contains(Facebook))

Reply:
<details>
<summary>Link to facebook group</summary>
<img src="https://user-images.githubusercontent.com/18536328/75995430-775cb780-5f0d-11ea-87c2-2a116c1fb33c.png" />
</details>